src/pybind/mgr/dashboard/frontend/src/app/core/auth/login-password-form/login-password-form.component.html

```diff
@@ -0,0 +1,102 @@
+<div class="login-password-form">
+  <h1 i18n>Please set a new password.</h1>
+  <h4 i18n>You will be redirected to the login page afterwards.</h4>
```

Comment:
With https://tracker.ceph.com/issues/43616 "mgr/dashboard: Keep the user logged in after password change" im wondering if we still want to redirect to the login-page after the password change?

Reply:
This is still necessary because after changing the password the next REST API call will return a 401 which will lead the Angular router to show the login page.

Reply:
I can prepare this PR to work like suggested in the tracker, but there need to be done more things to get it working.

I will use `this.router.navigate(['']);`instead of `this.authService.logout();`.

Reply:
See https://github.com/ceph/ceph/pull/32680/files#diff-f9b89437a4b191662fcf898eb97b6babR45